pom.xml

```diff
@@ -70,6 +69,15 @@
                 <module>../ignite</module>
             </modules>
         </profile>
+        <profile>
```

Comment:
I would suggest to not move the gatling-ext as a whole but the examples sub-module only.

It's the examples sub-module who use the gatling-maven-plugin which is built with jdk-11.

In other words modify the modules/gatling-ext/pom.xml  moving the examples to java-11+ profile as

```
    <modules>
        <module>gatling-plugin</module>
    </modules>

    <profiles>
        <profile>
            <id>java-11+</id>
            <activation>
                <jdk>[11,)</jdk>
            </activation>
            <modules>
                <module>examples</module>
            </modules>
        </profile>
    </profiles>
```

Reply:
Unfortunately, `gatling-plugin` still requires java11 to run tests, it can be compiled on java8, but can't be used and will be failed on TC.